config/crd/bases/cloud.redhat.com_clowdenvironments.yaml

```diff
@@ -279,6 +300,7 @@ spec:
                 this file'
               type: string
           required:
+          - apps
```

Comment:
I don't think it should be required, but I'm not sure where this comes from - it shouldn't be required because if there are no apps, this will not get populated. Will it just get initialized to an empty array?